### PR TITLE
Set proposal outcome

### PR DESCRIPTION
**Pull request by HSalmon**
screen shot 2016-02-23 at 2 38 52 pm

### CODE REVIEW

**Comment by HSalmon** [comment]
@rdbartlett what should this notification say?

**Comment by HSalmon** [comment]
Addresses https://github.com/loomio/loomio/issues/2990

File: angular/core/components/notifications/notifications.coffee [hunk 1]
```diff
@@ -13,6 +13,7 @@ angular.module('loomioApp').directive 'notifications', ->
       'membership_requested',
       'membership_request_approved',
       'user_added_to_group',
+      'motion_closed',
```
Comment (gdpelican) [inline angular/core/components/notifications/notifications.coffee]: Technical debt note: I'd like to move this list out of here and into `notification.rb`, alongside BETA_NOTIFICATIONS

File: angular/test/protractor/notifications_spec.coffee [hunk 2]
```diff
@@ -2,29 +2,19 @@ describe 'Notifications', ->
...
-    expect(notificationsDropdownText()).toContain('replied to your comment')
-    expect(notificationsDropdownText()).toContain('published an outcome')
-    expect(notificationsDropdownText()).toContain('Proposal is closing')
-    expect(notificationsDropdownText()).toContain('liked your comment')
+    page.loadPath 'setup_all_notifications'
+
+    page.expectText '.notifications__activity', '9'
+    page.click '.notifications__button'
+
+    page.expectText '.notifications__dropdown', 'approved your request'
```
Comment (gdpelican) [inline angular/test/protractor/notifications_spec.coffee]: Are we missing 'added you to the group' here?

**Comment by rdbartlett** [comment]
Thanks Hannah! How's this? "Proposal closed: Holiday on the moon. Publish an outcome."

**Comment by HSalmon** [comment]
screen shot 2016-02-24 at 10 12 41 am

**Comment by gdpelican** [comment]
I'm not a big fan of how that looks, but happy to have it go in knowing we're going to have a good hard look at the notifications later this year.

**Pull request merged**